### PR TITLE
update readme

### LINKED ISSUE #17: Readme out of date
- [x] readme starts with 
     >bifacialvf is a self-contained view factor (or configuration factor) model which replicates a 5-row PV system

     AFAICT this is no longer true, using `simulate` any system can be modeled
- [x] readme says this is only Python-2.7, but AFAICT it works with Python-3 fine

### PR DESCRIPTION
**Pull request by mikofski**
closes #17 
- remove 5-rows, is this still true? AFAICT only one row is simulated, determined by the `rowType` parameter
- update python version, works for 2 and 3
- update requirements, must have Numpy (for vf.py) and pvlib python for bifacialvf.py to do single-axis tracking.
- update installation from cheeseshop, gh only for dev mode
- some minor formatting changes, hopefully improvements?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,24 +15,29 @@ URL: https://www.nrel.gov/docs/fy17osti/67847.pdf
 ## Introduction
 
 bifacialvf is a self-contained view factor (or configuration factor)
-model which replicates a 5-row PV system of infinite extent perpendicular to the module rows. 
+model which replicates a single row of a PV system of infinite extent perpendicular to the module rows.
```
Comment (cdeline) [inline README.md]: This isn't true.  The code assumes 5 rows of PV - the middle (interior) row is simulated by default, but user interface options include 'first, interior, last, single'.  We could add a statement that these are the simulation options..

File: README.md [hunk 2]
```diff
@@ -15,24 +15,29 @@ URL: https://www.nrel.gov/docs/fy17osti/67847.pdf
...
 bifacialvf is a self-contained view factor (or configuration factor)
-model which replicates a 5-row PV system of infinite extent perpendicular to the module rows. 
+model which replicates a single row of a PV system of infinite extent perpendicular to the module rows. 
 Single-axis tracking is supported, and hourly output files based on TMY inputs 
 are saved.  Spatial nonuniformity is reported, with multiple rear-facing irradiances collected
 on the back of each module row.
 
 ## Pre-requisites
-This software is written for Anaconda Python 2.7 which can be downloaded here: https://www.anaconda.com/download/
+This software is written for Python 2 or 3. NREL recommends [Anaconda Python](https://www.anaconda.com/download/).
```
Comment (cdeline) [inline README.md]: yay, thanks!

**Comment by mikofski** [comment]
@cdeline thanks for your review and comments. I've reverted the 5-row edit, and added your clarifications.

**Comment by mikofski** [comment]
Also addresses #5 (update install instructions).

**Pull request merged**